controllers/events.js

```diff
@@ -3,7 +3,7 @@ var express = require('express'),
 var fs = require('fs');
 
 // API keys to access the skiddle festival database
-var contents = fs.readFileSync(__dirname + '/../public/json_dump/events_dump_23_2_16.json');
+var contents = fs.readFileSync(__dirname + '/../public/json_dump/smaller_dump_23_2_16.json');
```

Comment:
i made a json dump that has fewer events, but wanted to keep the full events dump if i needed